osf/migrations/0201_remove_prereg_waffling.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.13 on 2019-07-23 14:26
+from __future__ import unicode_literals
+from django.db import migrations
+
+from osf import features
+from osf.utils.migrations import DeleteWaffleFlags
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('osf', '0200_auto_20200214_1518'),
+    ]
+
+    operations = [
+        DeleteWaffleFlags([features.OSF_PREREGISTRATION]),
```

Comment:
hm, this migration (and 0144) is the only thing keeping us from deleting `features.OSF_PREREGISTRATION`, and we won't want migrations for adding/removing waffles once #9308 is merged -- wondering if we should remove the migration and the switch now, knowing it'll eventually get cleaned up... thoughts?

Reply:
Yeah, I want to wait to fix things up, it's likely all these migrations are going to be squashed soon anyway. I'd want to wait unitl #9308 is merged.